README.md

```diff
@@ -1,40 +1,96 @@
+    * `mustard`: Contains third-party vendor JS polyfills that will be pulled in if browser doesn't cut the mustard
+    * `app`: Custom written JS that includes common distributed and optional files. Supports ES6 syntax which will be transpiled by Babel during the build process.
+      *`common`: Custom written JS that is needed by DCF and will be packaged with and distributed with core
+      *`optional`: Optional custom written JS modules that can be required when additional functionality is wanted.
+      
+      
+#### CSS Files
+`/assets/src/css/vendor/**/*.css`
+DCF Core uses SASS. The CSS folder is mainly used for vendor associated CSS files 
+
 
-Run the following command to build the development files:
 
-```
-npm run dev -- --env.theme your_theme_name
-```
 
-for example: `npm run dev -- --env.theme example` will build the example theme that lives in `/theme/example`.
+## Gulp Tasks & Build Process
+DCF Core uses Gulp 4.0. Gulp tasks are specified in the `gulpfile.js`. To modularize commonly used variables and functions, they are specified in the `/build-utils` folder and required in the gulpfile as needed. This also help keep the gulpfile easy to scan and dry.
+
+
+### Build-utils Folder Contents
+* `banner.js` - contains copyright/owner info banner template that extracts information out of `package.json` and injected into files 
+* `build-names.js` - contains variable names of files outputted into the build folder
+* `build-paths.js` - contains path variables of files going that are being outputted into the `/assets/build` folder by Gulp
+* `cascade-delete.js` - custom written reusable function to delete similar files in either `/build` or `/dist` when files are being deleted from `/src` when the watch tasks are running
+* `check-directory.js` - custom function to check if a directory exists and if not create it, used in outputting eslint error logs
+* `common-paths.js` - contains base path variables to be composed from in build-paths.js and dist-paths.js
+* `concat.js` - contains object with two Gulp concat task functions, one with and one without gulp-newer
+* `custom-plumber.js` - error notification function using gulp-notify
+* `dist-names.js` - contains variable names of files outputted into the build folder
+* `dist-paths.js` - contains path variables of files going that are being outputted into the `/assets/build` folder by Gulp
+* `gulp-load-plugins.js` - contains function to pull in `package.json` plugins using gulp-load-plugins and store it in the `$` variable. Scoped to just devDependencies plugins.
+*  `uglify.js` - contains function to run gulp-uglify task, preserves comments that starts with @license or @preserve, also shows anticipated gzipped size 
+
+
+### Gulp Processing for SCSS, JS, CSS files
+ * SASS files - The src `_*.scss` partials are linted and then copied to the `assets/dist/scss` folder. Lint errors will be displayed in the console and also outputed to `/logs/stylelint` 
```

Comment:
If we are just linting, what good does it do to copy the files to the `src/` folder?

Reply:
Not quite following, will touch base with you on this.

Reply:
Talked about this, resolution is to empty the dist folder and have users run the default Gulp task to rebuild it so that the distribution is lighter.